ui-tests/src/test/resources/features/integrations/api-provider.feature

```diff
@@ -769,8 +841,8 @@ Feature: API Provider Integration
     And delete step on position 3
```

Comment:
This needs to be `position 2`. Don't ask how I know.

Reply:
Changed to position 2... how did you know?